packages/location-management/src/components/LocationTree/index.tsx

```diff
@@ -12,6 +12,7 @@ import reducer, {
 import { AntTreeProps } from '../LocationUnitView';
 import './tree.css';
 import { Dictionary } from '@onaio/utils';
+import { ParsedHierarchyNode } from 'location-management/src/ducks/locationHierarchy/types';
```

Comment:
Such imports from within a package have proven to raise errors, could we change this to a relative path i.e `import { ParsedHierarchyNode } from '../../ducks/locationHierarchy/types';`